packages/omgx/wallet-frontend/src/components/Proposal/Proposal.js

```diff
@@ -83,6 +83,9 @@ function Proposal({
                 if(proposal.state !== 'Active') {
                     return;
                 }
+                if(proposal.hasVoted) {
```

Comment:
With early return code at this lie will not get execute.

I mean the user won't be able to see the details text: I guess we need to remove check at the bottom or this early return.

https://github.com/omgnetwork/optimism/pull/434/files#diff-32b2019ca228a53023b0b623415eda8d05399a80c13edfa4a1338fd66c4e0cd3R134

Reply:
we only need to show the vote buttons IF (the proposal is active) && (the person has not yet voted) so it looks ok